pages/specification/migration/_index.md

```diff
@@ -3,5 +3,85 @@ title: Migrating from older drafts
+| `$defs`                     | Core          | 2019-09            | No      | Replaced `definitions`                                                                       |
+| `$dynamicAnchor`            | Core          | 2020-12            | No      | Replaced `$recursiveAnchor`                                                                  |
+| `$dynamicRef`               | Core          | 2020-12            | No      | Replaced `$recursiveRef`                                                                     |
+| `$recursiveAnchor`          | Core          | 2019-09            | 2020-12 | Replaced by `$dynamicAnchor`                                                                 |
+| `$recursiveRef`             | Core          | 2019-09            | 2020-12 | Replaced by `$dynamicRef`                                                                    |
+| `$ref` (legacy)             | Core          | 03                 | 2019-09 | Replaced by `$ref`                                                                           |
+| `$ref`                      | Core          | 2019-09            | No      | Replaced `$ref` (legacy)                                                                     |
+| `$schema`                   | Core          | 03                 | No      | No                                                                                           |
+| `$vocabulary`               | Core          | 2019-09            | No      | No                                                                                           |
+| `additionalItems`           | Core          | 03                 | 2020-12 | Replaced by `items`                                                                          |
+| `allOf`                     | Core          | 04                 | No      | No                                                                                           |
+| `anyOf`                     | Core          | 04                 | No      | No                                                                                           |
+| `const`                     | Validation    | 06                 | No      | No                                                                                           |
+| `contains`                  | Core          | 06                 | No      | No                                                                                           |
+| `contentEncoding` (legacy)  | Core          | 01                 | 04      | Replaced by `media`.`binaryEncoding`                                                         |
+| `media`                     | Hyper-Schema  | 04                 | 07      | Replaced `contentEncoding` (legacy) and replaced by `contentEncoding` and `contentMediaType` |
+| `contentEncoding`           | Validation    | 07                 | No      | Replaced `media`.`binaryEncoding`                                                            |
```

Comment:
I don't think the notes make sense. `contentEncoding` never replaced any other keyword. That said, on 2019-09, it became an annotation keyword